doc/man3/flux_request_decode.adoc

```diff
@@ -22,7 +22,7 @@ SYNOPSIS
 
  int flux_request_decode_raw (const flux_msg_t *msg,
                               const char **topic,
-                              void *data, int *len);
+                              void **data, int *len);
```

Comment:
I think the cast to `(void **)` is not so bad for increased clarity in the prototype.

If it becomes too onerous a user can create a wrapper function so at most we'd see the case once per file/module.